frontend/src/components/pages/ReviewPage.tsx

```diff
@@ -85,81 +93,101 @@ const ReviewPage = () => {
     },
   });
 
-  return (
-    <Flex
-      height="100vh"
-      direction="column"
-      position="absolute"
-      top="0"
-      bottom="0"
-      left="0"
-      right="0"
-      overflow="hidden"
-    >
-      <Header title={title} />
-      <Divider />
-      <Flex justify="space-between" flex={1} minHeight={0}>
-        <Flex width="100%" direction="column">
-          <Flex justify="space-between" alignItems="center" margin="10px 30px">
-            <FontSizeSlider setFontSize={handleFontSizeChange} />
-          </Flex>
-          <Divider />
+  const ReviewPageContent = () => {
+    return (
+      <div>
```

Comment:
nit: you could use a[ react fragment](https://reactjs.org/docs/fragments.html#short-syntax) here using `<></>`